server/src/main/resources/db/migration/V1_325_add_dulingo_coupon_task.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024 Talent Beyond Boundaries.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+insert into task (admin, created_by, created_date, days_to_complete, description, name, optional, task_type,
+                  display_name)
+values (false, 25043, now(), 14, 'We are offering you a free Duolingo Test.', 'duolingoTest', false, 'Task',
```

Comment:
Hi @MuhannedNoman, just while I'm on your PR I see you've used id 25043 for this created_by task field, what user does that id belong to? We have created past tasks using the System Admin user, if you see flyway # 167 for example you can see we do "(select id from users where username = 'SystemAdmin')," for the created_by field on other tasks.